src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -157,6 +157,30 @@ ${listener}.addEventListener(
         setAutoScroll();
       }
     }
+      // Handle sync scroll to sequence
+    if (message.hasOwnProperty("action") && message.action === "scrollToSequence") {
+      const element = document.getElementById(message.sequence);
```

Comment:
The message.sequence can be sanitized and validated before using it directly in DOM object. This would protect from XSS vulnerability.

Possible fix
- Validate if this message.sequence only contains a specific pattern using regex

```javascript
// this regex can change based on what is expected here
const ALLOWED_SEQUENCE_PATTERN = /^[a-zA-Z0-9\-_]+$/; 

if (message.hasOwnProperty("action") && message.action === "scrollToSequence") {
  const sequence = message.sequence;
  
  if (!sequence || !ALLOWED_SEQUENCE_PATTERN.test(String(sequence))) {
    console.warn('Invalid sequence format:', sequence);
    return;
  }
  
  const element = document.getElementById(sequence);
  // ... rest of the code
}
```